Examples/BinaryBH/BinaryBHLevel.hpp

```diff
@@ -6,9 +6,9 @@
 #ifndef BINARYBHLEVEL_HPP_
 #define BINARYBHLEVEL_HPP_
 
-#include "BHAMR.hpp"
```

Comment:
I guess we should still keep this include for the non TP case?

Reply:
It's included via TPAMR.hpp.

Reply:
Yeah I see, but just for aesthetics, no? :-)

Reply:
I actually originally had it but removed it as I prefer it this way. I can add a comment though.